kairon/api/app/routers/bot/integrations.py

```
@@ -0,0 +1,84 @@
+from kairon.shared.constants import CatalogProvider
+from kairon.shared.constants import DESIGNER_ACCESS
+from kairon.shared.models import User
+
+router = APIRouter()
+cognition_processor = CognitionDataProcessor()
+
+@router.post("/{provider}/{sync_type}/{bot}/{token}", response_model=Response)
+async def sync_data(
+    request: Request,
+    provider: CatalogProvider = Path(description="Catalog provider name",
+                                 examples=[CatalogProvider.PETPOOJA.value]),
```

Ruff (0.8.2)

Do not perform function call `Path` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)